get-env.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
if u meant `set -e`, i did not want the script to exit on failure by design, considering the map of commands it iterates through, i wanted the least friction as possible considering the variety of systems this could be run on. But to mention your other point, probably a good idea to keep it informative, ie output any stderr without exiting.